src/components/Login.css

```diff
@@ -4,7 +4,8 @@
   left: 50%;
   transform: translate(-50%, -50%);
   width: 400px;
-  background: white;
+  background: wh
+  ite;
```

Comment:
it should  be in one line like background: white; but its better practice to use rgb for example :  use #fff for white and #000 for black.

Reply:
done.